src/tools/Scoop/Import-ScDatabases.ps1

```diff
@@ -83,6 +83,12 @@ Function Import-ScDatabases
             mkdir $myTemp | Out-Null
         }
 
+        $DatabaseDbPrefix = $config.DatabaseDbPrefix;
+        $BackupPrefix = $config.DatabaseBackupPrefix;
+        if(-not $BackupPrefix) {
+            Write-Verbose "The backups prefix could not be found. Default database prefix '$($DatabaseDbPrefix)' will be used."
```

Comment:
DatabaseDbPrefix will proabaly only be set, if DatabaseBackupPrefix is set, so the message is a bit useless. Rather say that the database name will be used to find the correct backup